internal/kafka/command_broker_update.go

```diff
@@ -38,5 +38,5 @@ func (c *brokerCommand) update(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	return broker.Update(cmd, args, restClient, restContext, clusterId)
+	return broker.Update(cmd, args, restClient, restContext, clusterId, c.Config.EnableColor)
```

Comment:
Printing should only happen in `internal/`. We shouldn't be passing the enable colors config to `pkg/` since that's exposed to the outside world, and it would be weird if you imported a package that printed to stdout as a side-effect.

Reply:
I moved the printing part of `broker.Update` out of that function and into the two commands that use it.